### PR TITLE
Differentiate test/run configurations in statistics

### PR DESCRIPTION
**Pull request by wbars**

### CODE REVIEW

**Review by Undin — COMMENTED** [review]
Wow!
Nice to see that we actually can pass additional info with run configurations. Didn't know about `FusAwareRunConfiguration` before

BTW it would be nice to collect targets used for build/run. Not a request for this PR

File: src/main/kotlin/org/rust/ide/statistics/CargoCommandUsagesCollector.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.statistics
+
+import com.intellij.execution.impl.statistics.RunConfigurationUsageTriggerCollector
+import com.intellij.internal.statistic.eventLog.events.EventField
```
Comment (mchernyavsky) [inline src/main/kotlin/org/rust/ide/statistics/CargoCommandUsagesCollector.kt:9]: Unused import

**Review by mchernyavsky — APPROVED** [review]
LGTM

**Comment by wbars** [comment]
bors r=mchernyavsky

**Pull request merged**